Lib/test/test_getopt.py

```diff
@@ -19,21 +19,34 @@ def assertError(self, *args, **kwargs):
         self.assertRaises(getopt.GetoptError, *args, **kwargs)
 
     def test_short_has_arg(self):
-        self.assertTrue(getopt.short_has_arg('a', 'a:'))
-        self.assertFalse(getopt.short_has_arg('a', 'a'))
+        self.assertIs(getopt.short_has_arg('a', 'a:'), True)
```

Comment:
using assertIs for True and False checks is unusual and too specifically tied to the implementation detail that those happen to be singletons.  Add comments anywhere you do this as to _why_ assertTrue or even assertEqual would be incorrect.

A nicer non-implementation detail tied way to express this that is probably what you were intending is: assertIsInstance(thing, bool) + assertTrue/False

Reply:
This is a change from #126375. It should be reviewed there.